### ISSUE TITLE
modifying ion-content's padding css variables also alters fab-button padding

### ISSUE THREAD

**Issue by zakton5** [ionic-team/ionic#17045]
# Bug Report

**Ionic version:**

[x] **4.x**

**Current behavior:**

Specifically, I'm attempting to modify `--padding-bottom` on `ion-content` so that the `ion-fab` does not cover my list items when the user scrolls to the bottom of the list. Doing so sets the `--padding-bottom` of the content and the fab.

**Expected behavior:**
It should only modify the padding of the content and not the fab

**Related code:**

```html
<ion-content fab-padding>
<ion-fab vertical="bottom" horizontal="end" slot="fixed">
        <ion-fab-button>
            <ion-icon name="add" class="add-icon-custom"></ion-icon>
        </ion-fab-button>
</ion-fab>
</ion-content>
```

```scss
ion-content[fab-padding] {
  --padding-bottom: 72px;
}
```

Results in this:
<img width="108" alt="screen shot 2019-01-10 at 2 43 37 pm" src="https://user-images.githubusercontent.com/7013396/50992853-32deb300-14e6-11e9-9da6-4d4994ed912b.png">

**Other information:**

**Ionic info:** 

```
Ionic:

   ionic (Ionic CLI)             : 4.7.1 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.0.0-rc.1
   @angular-devkit/build-angular : 0.11.4
   @angular-devkit/schematics    : 7.1.4
   @angular/cli                  : 7.1.4
   @ionic/angular-toolkit        : 1.2.0

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : none
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, (and 7 other plugins)

System:

   ios-deploy : 1.9.4
   ios-sim    : 7.0.0
   NodeJS     : v11.5.0 (/usr/local/Cellar/node/11.5.0/bin/node)
   npm        : 6.5.0
   OS         : macOS Mojave
   Xcode      : Xcode 10.1 Build version 10B61
```

**Comment by paulstelzer** [comment]
You are right because the fab button has no --padding-bottom value set, so he gets the value from the parent (in your case the ion content).

The fab button has to get default value for --padding-bottom and co

**Comment by manucorporat** [comment]
Fixed by #17045!! Thanks for the issue!
